### ISSUE TITLE
Is the training code working? If not is anyone working on getting it to work?

### ISSUE THREAD

**Issue by adiRpatgaonkar** [multimodallearning/pytorch-mask-rcnn#9]

**Comment by lasseha** [comment]
Yes, you can either start from pretrained imagenet or coco weights.

**Comment by adityarp9** [comment]
Okay, I'll try that. Thanks.

**Comment by adityarp9** [comment]
Also, since this is a single GPU training, do I need to change the hyper-parameters to get the AP specified in your Readme?

**Comment by lasseha** [comment]
With the current schedule i could not get the AP of the matterport implementation, so i assume we need to adjust the hyper-parameters and the training schedule.

**Comment by adityarp9** [comment]
Okay cool. I will try changing the hyper-parameters to get Matterport's
mAP. Will report accordingly.

**Comment by adityarp9** [comment]
Could you please share the mAP that you are getting with the current hyperparameters?

**Comment by lasseha** [comment]
I was getting mAP of 26.5 (bbox) and 22.3 (segm).

**Comment by choltz95** [comment]
Got 37.3 bbox and 32.5 segm by incorporating the maskx rcnn extension from cvpr this year & early stopping...no change to the dataset like the paper, but I doubled the default training regime.

**Comment by adityarp9** [comment]
Okay cool. Closing issue.
